test/endpoints/server-side.js

```diff
@@ -215,6 +225,9 @@ describe('endpoints#serverSide', function() {
   });
 
   it('should throw on server side queue timeout error', function(done) {
+
+    console.log(' ');
```

Comment:
What happens if you do a `this.timeout(2500);` rather than this console statement? Does it work then?